pkg/crc/api/api.go

```diff
@@ -117,7 +119,7 @@ func (api Server) handleConnections(conn net.Conn) {
 			conn.Close()
 		}
 
-	case "status", "version", "setconfig", "getconfig", "unsetconfig", "webconsoleurl", "logs":
+	case "status", "version", "setconfig", "getconfig", "unsetconfig", "webconsoleurl", "logs", "ready":
```

Comment:
slightly worried at running multiple `ready` in parallel even if it should be fine.